Dockerfile

```diff
@@ -1,36 +1,43 @@
-FROM registry.access.redhat.com/ubi8/ubi-minimal as builder
-
-# create enviroment variables and files
-ENV ENVFILE=/tmp/envfile
-RUN echo "export ARCH=`uname -m | sed 's/x86_64/amd64/'`" >> ${ENVFILE}
-ENV PATH=$PATH:/usr/local/go/bin
+ARG TARGET_ARCH=amd64
+FROM registry.access.redhat.com/ubi8/ubi-minimal as base
+ARG TARGET_ARCH
```

Comment:
Is it necessary to redefine `TARGET_ARCH` here?

Reply:
Yes, it works with my Podman without any issues. However, it appears that in the container engine used in the CI, it does not work as expected. Essentially, if we do not reactivate the ARG in every stage, it will not be readable in the FROM statement on line 34. It took me a while to figure out why the tests were failing.